### PR TITLE
Styling improvements

### PR DESCRIPTION
**Pull request by lachlanjc**

### CODE REVIEW

File: src/components/chat/chat_input.jsx [hunk 1]
```diff
@@ -21,15 +21,19 @@ const ChatInput = React.createClass({
       user
     } = this.state;
 
+    let style = {
+      boxShadow: '-4px 0 15px rgba(0,0,0,.15)'
+    }
+
     if (user.get('username')) {
-      return (
+      return (<div className="full-width shadow px3 py1" style={style}>
```
Comment (pletcher) [inline src/components/chat/chat_input.jsx]: Nit: newline after opening paren and before closing paren.

Comment (lachlanjc) [inline src/components/chat/chat_input.jsx]: Lol ok :)

**Comment by lachlanjc** [comment]
@pletcher :)

**Comment by pletcher** [comment]
All right, :LGTM:

**Pull request merged**